### PR TITLE
60 set default language to be <by>

### PR DESCRIPTION
**Pull request by dmitrap**

### CODE REVIEW

File: src/components/EntryForm.jsx [hunk 1]
```diff
@@ -49,9 +49,8 @@ class Form extends Component {
                 <FieldElement className="pure-input-1" name="category" component="select">
                   <option value={-1}>- {t("chooseCategory")} -</option>
                   <option value={IDS.INITIATIVE}>{t("category." + NAMES[IDS.INITIATIVE])}</option>
-                  {/* this functionality will be made in future */}
-                  {/* <option value={IDS.COMPANY}>{t("category." + NAMES[IDS.COMPANY])}</option> */}
-                  <option value={IDS.EVENT}>{t("category." + NAMES[IDS.EVENT])}</option>
```
Comment (KonstantinKudelko) [inline src/components/EntryForm.jsx]: @dmitrap why do you commit the `event` option?

Comment (dmitrap) [reply src/components/EntryForm.jsx]: nevermind that

Comment (KonstantinKudelko) [reply src/components/EntryForm.jsx]: thx for revert chagnes

**Comment by fr0zen** [comment]
@dmitrap pushing of file translation-by.json here will destroy actual translation. Please exclude it

**Pull request closed without merging**